modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsStructure.testcase

```diff
@@ -25,13 +25,28 @@ definition {
 	}
 
 	@description = "This is a test for LPS-123536. As Content Creator, I want to define reusable translations when creating WEM Structures with nested field"
-	@ignore = "true"
 	@priority = "4"
 	test CheckDefaultStructureTitleDisplayedOnStructureList {
-		property portal.acceptance = "false";
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(structureName = "WC Structure Name");
 
-		// TODO LPS-123536 CheckDefaultStructureTitleDisplayedOnStructureList pending implementation
+		DataEngine.addField(
+			fieldFieldLabel = "Text",
+			fieldName = "Text");
 
+		DataEngine.addFieldNested(
+			fieldFieldLabel = "Numeric",
+			fieldName = "Numeric",
+			targetFieldLabel = "Text");
+
+		WebContentStructures.saveCP(structureName = "WC Structure Name");
+
```

Comment:
Hey @gustavolimav , I think would be good to add a second translation to the structure before saving it. Then validate if the default structure title is shown in the list.